tensorflow/python/ops/array_ops.py

```diff
@@ -3157,41 +3157,59 @@ def edit_distance(hypothesis, truth, normalize=True, name="edit_distance"):
   You can normalize the edit distance by length of `truth` by setting
   `normalize` to true.
 
-  For example, given the following input:
+  For example:
+
+  Given the following input,
+  * `hypothesis` is a `tf.SparseTensor` of shape `[2, 1, 1]`
```

Comment:
```...shape `[2, 1, 1]` ```
tensor shapes are tuples not lists. change every occurrence similar to the above to ```...shape `(2, 1, 1)` ```